src/main/java/com/mraof/minestuck/data/MinestuckEnUsLanguageProvider.java

```diff
@@ -1405,6 +1406,8 @@ protected void addTranslations()
 		add("message.dice.3", "You rolled a 4.");
 		add("message.dice.4", "You rolled a 5.");
 		add("message.dice.5", "You rolled a 6.");
+		
+		add("punch_designix.code_rejected", "On closer inspection, the code on the back seems to be almost unreadable");
```

Comment:
Create a public static final value in PunchDesignixBlockEntity with the location name. You can add it after line 1748 in this file, modeling it off of the RedstoneClockBlockEntity example that exists on line 1748

Reply:
done